extensions/laravel-forge/src/Site.tsx

```diff
@@ -81,6 +81,18 @@ export const SitesSingleView = ({ site, server }: { site: ISite; server: IServer
     <>
       <List searchBarPlaceholder="Search sites...">
         <List.Section title={`Site Commands (${current.name})`}>
+          <List.Item
+            id="open-on-forge"
+            key="open-on-forge"
+            title="Open on Laravel Forge"
+            icon={Icon.Globe}
+            accessoryTitle="forge.laravel.com"
```

Comment:
This should probably have the site name included, right?

Reply:
I guess it could; I made it parallel with the server but 🤷‍♂️

Reply:
It's fine either way. What ever you think. I just added it originally because it looked worse with nothing there at all. It's probably okay how you have it now.